codespeed/mercurial.py

```diff
@@ -45,8 +45,7 @@ def getlogs(endrev, startrev):
     updaterepo(endrev.branch.project, update=False)
 
     cmd = ["hg", "log",
-            "-r", "%s:%s" % (endrev.commitid, startrev.commitid),
-            "-b", "default",
```

Comment:
When using the :: syntax with two revisions on the same branch, there is no need to filter by branch.